### DISCUSSION TITLE
is fzyzcjy.github.io/flutter_rust_bridge/  dead?😢

### DISCUSSION THREAD

**Discussion by MouriKogorou** [fzyzcjy/flutter_rust_bridge#2366] [General]
[fzyzcjy.github.io/flutter_rust_bridge/](https://fzyzcjy.github.io/flutter_rust_bridge/) website can't access, please check.

**Comment by fzyzcjy** [comment]
Checked, I can visit it. Maybe because your network cannot visit github.io?

![image](https://github.com/user-attachments/assets/9b25c3d7-7068-42a8-8460-68d1c8c3b8d2)

**Reply by MouriKogorou** [reply]
thanks, it's my network problem

**Reply by fzyzcjy** [reply]
You are welcome!
